docs/rules/arrow-parens.md

```diff
@@ -0,0 +1,67 @@
+# Require parens in arrow function arguments (arrow-parens)
```

Comment:
Please share some notes as to why people should do this way.
And also under the rule detail section, mention the options.